packages/server-core/src/projects/githubapp/githubapp-helper.ts

```diff
@@ -81,14 +81,16 @@ export const getGitHubAppRepos = async () => {
   }
 }
 
-export const getAuthenticatedRepo = async (repositoryPath) => {
+export const getAuthenticatedRepo = async (repositoryPath: string, reset = false) => {
   try {
+    console.log('getAuthenticatedRepo', repositoryPath, reset)
```

Comment:
Do we want to leave the console.logs in here, or convert to logger statements?

Reply:
I removed all the console.log's I'd put in for testing it.